internal/cmd/args.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// ExactArgs validates that a cobra command is executed with the exactly
+// number of command line arguments, otherwise it returns an error that includes
+// the usage string.
+func ExactArgs(number int) cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		if len(args) == number {
+			return nil
+		}
+		return fmt.Errorf(
+			"%q requires exactly %d %s.\nSee '%s --help'.\n\nUsage:  %s\n",
+			cmd.CommandPath(),
+			number,
+			pluralize("argument", number),
+			cmd.CommandPath(),
+			cmd.UseLine())
```

Comment:
Example:
```
ERROR  "infra grants add" requires exactly 2 arguments.
See 'infra grants add --help'.

Usage:  infra grants add IDENTITY DESTINATION [flags]

```

Reply:
Hmm, maybe this should use only single quotes or only double quotes, instead of both